src/sky.py

```diff
@@ -22,6 +22,9 @@ def __init__(self, game_time):
             '20' : (255, 219, 203),
             '24' : (38, 101, 189)
         }
+        self.colors_hours = list(self.colors.keys)
```

Comment:
Slight oversight here: this will cause an error, because you forgot to put the parentheses when calling `self.colors.keys`. So right now you're actually calling the initialiser of the list class on _the bound method object representing `self.colors.keys`_, not the result of calling `self.colors.keys()`